pkg/apis/core/v1alpha1/uibutton_types.go

```diff
@@ -253,6 +253,23 @@ type UIHiddenInputStatus struct {
 	Value string `json:"value" protobuf:"bytes,1,opt,name=value"`
 }
 
+// Describes a choice dropdown input field attached to a button
+type UIChoiceInputSpec struct {
+	// Initial value for this field.
+	//
+	// +optional
+	DefaultValue string `json:"defaultValue" protobuf:"bytes,1,opt,name=defaultValue"`
```

Comment:
is it ok to have a DefaultValue that's not an element of the Choices array?

alternatively, what do you think of getting rid of DefaultValue, and making the default value implicitly the first Choice?

Reply:
Yeah now that you mention it, that seems like a way better solution.  I'll get rid of the DefaultValue